components/form/form-errory-summary.js

```diff
@@ -26,7 +26,6 @@ class FormErrorSummary extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			}
 
 			.d2l-form-error-summary-header {
-				align-items: top;
```

Comment:
`top` is not a supported value for `align-items` and this looked fine without so I didn't replace it with a different value.